app/page.tsx

```diff
@@ -1,54 +1,58 @@
+      {/* Top Nav Bar */}
+      <div className="absolute top-0 w-screen 2xl:h-28 h-20 flex flex-row text-wrap">
+        {/* Minted Logo */}
+        <div className="2xl:pt-6 pt-4 2xl:pl-10 pl-6"><Link href="/" className={`${caveat.className} 2xl:text-6xl text-5xl text-black font-medium`}>Minted</Link></div>
+        {/* Header Tabs */}
+        <div className="2xl:px-40 px-20 2xl:space-x-20 space-x-12 flex flex-row text-2xl text-heading items-center">
+          <Link href="/about">About</Link>
+          <Link href="/how_it_works">How It Works</Link>
+          <Link href="/why_minted">Why Minted</Link>
+        </div>
+        {/* Log In and Dashboard Button */}
+        <div className="flex flex-row flex-1 text-heading justify-end items-center">
+          {email && <div className="2xl:pr-10 pr-6 text-xl font-medium">Hey, {email}!</div>}
+          {email ?
+          <Link href="/dashboard" className="2xl:mr-10 mr-6 2xl:h-16 h-12 2xl:w-40 w-32 border-4 border-darkmaroon border-solid rounded-full text-darkmaroon 2xl:text-xl text-lg font-semibold flex justify-center items-center hover:bg-darkmaroon hover:text-primary transition duration-300">Dashboard</Link>
+          :
+          <Link href="/login" className="2xl:mr-10 mr-6 2xl:h-16 h-12 2xl:w-40 w-32 border-4 border-darkmaroon border-solid rounded-full text-darkmaroon 2xl:text-xl text-lg font-semibold flex justify-center items-center hover:bg-darkmaroon hover:text-primary transition duration-300">Log In</Link>
+          }
         </div>
-      </nav>
-
-      <div className="animate-in flex-1 flex flex-col gap-20 opacity-0 max-w-4xl px-3">
-        <Header />
-        <main className="flex-1 flex flex-col gap-6">
-          <h2 className="font-bold text-4xl mb-4">Next steps</h2>
-          {isSupabaseConnected ? <SignUpUserSteps /> : <ConnectSupabaseSteps />}
-        </main>
       </div>
 
-      <footer className="w-full border-t border-t-foreground/10 p-8 flex justify-center text-center text-xs">
-        <p>
-          Powered by{" "}
-          <a
-            href="https://supabase.com/?utm_source=create-next-app&utm_medium=template&utm_term=nextjs"
-            target="_blank"
-            className="font-bold hover:underline"
-            rel="noreferrer"
-          >
-            Supabase
-          </a>
-        </p>
-      </footer>
+      {/* Center Text */}
+      <div className="relative space-y-8 flex flex-col text-center">
+        <div className="space-y-8 2xl:text-8xl text-7xl text-darkmaroon font-semibold">
+          <div>Built for Non-Profits</div>
+          <div>Built for Change</div>
+        </div>
+        <div className="flex justify-center">
+          <div className="w-1/2 2xl:text-3xl text-2xl text-heading font-medium">Find opportunities to fund projects or start funding non-profits. We’re here to boost projects.</div>
```

Comment:
Font weight too heavy for smaller screens. Decrease it by one notch.

Reply:
This is meant for the description body only GH keeps showing previous lines of my selected line for reviews